README.md

```diff
@@ -146,7 +146,7 @@ require('telescope').setup{
     selection_strategy = "reset",
     sorting_strategy = "descending",
     layout_strategy = "horizontal",
-    layout_defaults = {
+    layout_config = {
```

Comment:
if we make this change, we should error in the setup if `layout_defaults` is passed. otherwise uses will be confused.